src/components/include/cbuf.h

```diff
@@ -224,7 +220,7 @@ cbuf_unpack(cbuf_t cb, u32_t *cbid)
 }
 
 static inline cbuf_t 
-cbuf_cons(u32_t cbid)
+cbuf_cons(int cbid)
```

Comment:
again, I'm not seeing the reasoning behind this.  I'm sure it is there, but I don't understand.